### PR TITLE
Kudzu fixes

### PR DESCRIPTION
**Pull request by Wunarg**
Various Kudzu changes
-Cost increased to 500 Wood, 200 dirt, 1 grain and 1 mithril ingot.
--These more complicated ressources heavily decreased how easily it can be spammed
-Increased Badger hole time build cost for the plant
-Sound effects for on hit and even particles for the blobs
-No longer grows into "no build zones" aka rails
-Takes a bit longer to get to get to its base max growth speed after beeing placed

### CODE REVIEW

**Comment by Wunarg** [comment]
Placing kudzu leaves (the mostly decorative ones) also gives 2 coins the same way placing nearly any block gives coins

**Comment by Wunarg** [comment]
Damaged kudzu is now brown on the minimap, this allows you to more easily tell if a green patch on the map is alive or just burned remains

File: TFlippy_TerritoryControl_Misc_Dev/Entities/Misc/Kudzu/KudzuHit.as [hunk 1]
```diff
@@ -19,12 +19,20 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
...
+			{
+				cuttingEffects(this, damage);
+			}
 			break;
 
 		case Hitters::spikes:
 		case Hitters::sword:
 		case Hitters::stab:
 			dmg *=  1.50f;
+			if(isClient())
```
Comment (Vam-Jam) [inline TFlippy_TerritoryControl_Misc_Dev/Entities/Misc/Kudzu/KudzuHit.as]: should be if (

File: TFlippy_TerritoryControl_Misc_Dev/Entities/Misc/Kudzu/KudzuHit.as [hunk 2]
```diff
@@ -19,12 +19,20 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
 	{
 		case Hitters::builder:
 			dmg *= 2.00f;
+			if(isClient())
```
Comment (Vam-Jam) [inline TFlippy_TerritoryControl_Misc_Dev/Entities/Misc/Kudzu/KudzuHit.as]: same here

**Review by Vam-Jam — APPROVED** [review]

File: TFlippy_TerritoryControl_Misc_Dev/Entities/Misc/Kudzu/KudzuHit.as (hunk 2 above)
Comment (Wunarg) [reply TFlippy_TerritoryControl_Misc_Dev/Entities/Misc/Kudzu/KudzuHit.as]: darn i copied the isClient() from base kag

**Pull request merged**